### PR TITLE
Fix compatibility issues with 2201.8.0

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
# Description
$subject. 

Fixes https://github.com/ballerina-platform/ballerina-extended-library/issues/577

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: ballerina/utils.bal [hunk 1]
```diff
@@ -538,10 +538,10 @@ isolated function copyFile(http:Client httpClient, string fileId, CopyFileOption
 # 
 # + httpClient - The HTTP Client
 # + fileId - ID of the file to be updated
+# + fileResource - 'FileMetadata' can be added as a payload to change metadata
 # + optional - 'UpdateFileMetadataOptional' used to add query parameters to the request
-# + fileResource - 'File' can added as a payload to change metadata
 # + return - If successful, returns `File`. Else returns `error`
-isolated function updateFileById(http:Client httpClient, string fileId, File? fileResource = (),
```
Comment (ayeshLK) [inline ballerina/utils.bal:544]: This seems like a breaking change, is this intentional ?

Comment (NipunaRanasinghe) [reply ballerina/utils.bal:544]: @ayeshLK this is just an util function consumed by the connector API. As this function is not public, won't be a breaking change IMO.

Comment (ayeshLK) [reply ballerina/utils.bal:544]: agreed

**Review by ayeshLK — COMMENTED** [review]
LGTM

**Review by ayeshLK — APPROVED** [review]

**Pull request merged**